types/sinon/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Sinon 9.0
+// Type definitions for Sinon 10.0
```

Comment:
It's very likely the right thing. But bumping the major version also implies that the rest of these types are now 10.0. Is that the case? If it's not then this will need a followup PR pretty quickly to make it so.

Reply:
it _should_ be, but i will double check. i think the versions were already diverged and 9 of the types was actually 10 of the code. ill make sure though and get back to you